### DISCUSSION TITLE
Config error in file

### DISCUSSION THREAD

**Discussion by ltlMirai** [end-4/dots-hyprland#628] [Q&A]
I get this, I just turned on transparency and I get Config error in file /home/user/.config/hypr/hyprland/colors.conf at line 9:rgba()expects length of 8 characters (4 bytes) or 4 comma separated values

Config error in file /home/user/hypr/hyprland.conf at line 9:Config error in file /home/user/.config/hypr/hyprland/colors.conf at line 9:rgba()expects length of 8 characters (4 bytes) or 4 comma separated values

I've ran this $ sass ~/.config/ags/scss/main.scss ~/.cache/ags/user/generated/style.css

Error: Can't find stylesheet to import.
  ╷
8 │ @import 'material'; // Material colors
  │         ^^^^^^^^^^
  ╵
  .config/ags/scss/main.scss 8:9  root stylesheet

**Comment by Simshaffer80** [comment]
I am having the exact same issue... :(
